deep_water_level/src/data.py

```diff
@@ -131,10 +131,10 @@ def get_data_loaders(
     # Create PyTorch DataLoaders for train and test splits
     return (
         torch.utils.data.DataLoader(
-            train_dataset, batch_size=batch_size, shuffle=True, normalize_output=normalize_output
+            train_dataset, batch_size=batch_size, shuffle=True,
```

Comment:
I don't know how this ever worked - DataLoader complains that it doesn't expect "normalize_output" as an argument, which makes sense. Am i missing something?

Reply:
I had the same question.  But anyway, we're not using cooc anymore for this, so I think it's ok to remove, and also remove it everywhere from this file, no?

Reply:
`cooc` = `coco`? Whether we use coco or csv doesn't affect this line of code, does it?